qemu/tests/cfg/blockdev_mirror_install.cfg

```diff
@@ -63,7 +63,7 @@
         qcow2:
             nbd_port_image1 = 10820
         raw:
-            nbd_port_image1 = 10819
```

Comment:
10819 is also used for a raw nbd image:
local image format: raw
export format: raw
VS.
10830:
local image format: qcow2
export format: qcow2

Not sure which is popular use in product, here clarify it, just for your reference.